app/views/start/index.html.slim

```diff
@@ -25,26 +25,18 @@
     h3.govuk-heading-s = t(".self_employment")
     p.govuk-body = t(".unacceptable_client_explanation")
   h3.govuk-heading-s = t(".unacceptable_client_heading_under_18")
-  - if FeatureFlags.enabled?(:mtr_phase_1, without_session_data: true)
-    p.govuk-body = t(".client_under_18_para_1")
-    p.govuk-body = t(".client_under_18_para_2")
-    ul.govuk-list.govuk-list--bullet
-      - t(".client_under_18_bullets").each do |bullet|
-        li = bullet
-  - else
-    p.govuk-body = t(".unacceptable_client_explanation_under_18_controlled_html",
-                   page_url: GuidanceLinkService.call(document: :lc_guidance_controlled,
-                                                      sub_section: :children),
-                   page_number: GuidanceLinkService.call(document: :lc_guidance_controlled,
-                                                         sub_section: :children,
-                                                         page_number_only: true))
+  p.govuk-body = t(".client_under_18_para_1")
+  p.govuk-body = t(".client_under_18_para_2")
+  ul.govuk-list.govuk-list--bullet
+    - t(".client_under_18_bullets").each do |bullet|
+      li = bullet
 
-    p.govuk-body = t(".unacceptable_client_explanation_under_18_certificated_html",
-                   page_url: GuidanceLinkService.call(document: :lc_guidance_certificated,
-                                                      sub_section: :children),
-                   page_number: GuidanceLinkService.call(document: :lc_guidance_certificated,
-                                                         sub_section: :children,
-                                                         page_number_only: true))
+  p.govuk-body = t(".unacceptable_client_explanation_under_18_certificated_html",
```

Comment:
This looks buggy to me. This paragraph used to be in the `else` block, so _not_ shown when mtr phase 1 was enabled. I think we want to remove this paragraph entirely now (and delete the key from en.yml as well).

Reply:
thanks, done